tests/FunctionalConnectorTest.php

```diff
@@ -4,12 +4,24 @@
 
 use Clue\React\Block;
 use React\EventLoop\Factory;
+use React\Socket\ConnectionInterface;
 use React\Socket\Connector;
+use React\Socket\ConnectorInterface;
 use React\Socket\TcpServer;
 
 class FunctionalConnectorTest extends TestCase
 {
-    const TIMEOUT = 1.0;
+    const TIMEOUT = 30.0;
+
+    private $ipv4 = false;
+    private $ipv6 = false;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->ipv4 = !!@file_get_contents('http://ipv4.tlund.se/');
+        $this->ipv6 = !!@file_get_contents('http://ipv6.tlund.se/');
+    }
```

Comment:
Not a fan of relying on internet connectivity here. Can this be avoided somehow?

If this is really needed, it should at least be behind the flag introduced via #146.

Reply:
Not unless we can run a local DNS server and socket server on different IP's. So I've added the internet group to those tests.

Reply:
The `file_get_contents()` will be executed before running any of the tests in this class, so each test now requires a working internet connection. Perhaps we can move this to a `@depends` definition or a separate class so it only affects *some* of the tests?

I don't want to block this for longer than needed, so I'm okay with relying on internet connectivity for now, but this should eventually be refactored IMHO. If this involves running a DNS server (mock), I'm fine with this :+1:

Reply:
> The `file_get_contents()` will be executed before running any of the tests in this class, so each test now requires a working internet connection. Perhaps we can move this to a `@depends` definition or a separate class so it only affects _some_ of the tests?

Good point, will change it.

> I don't want to block this for longer than needed, so I'm okay with relying on internet connectivity for now, but this should eventually be refactored IMHO. If this involves running a DNS server (mock), I'm fine with this

Agreed, but for now this does the trick.